.watchmanconfig

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Can we use this file to tell Watchman to ignore `target/`?

Reply:
I use watchman in combination with buck2; here's a decent starting point for what you might want to crib, which I came to after some fiddling: https://github.com/thoughtpolice/buck2-nix/blob/main/.watchmanconfig

Docs here: https://facebook.github.io/watchman/docs/config